ddtrace/contrib/wsgi/wsgi.py

```diff
@@ -132,6 +132,9 @@ def blocked_view():
                 core.dispatch("wsgi.request.prepare", (ctx, start_response))
                 try:
                     closing_iterable = self.app(environ, ctx.get_item("intercept_start_response"))
+                # StopIteration should not mark span with error: https://github.com/miguelgrinberg/flask-sock/issues/64
+                except StopIteration:
+                    core.dispatch("wsgi.app.success", (ctx, closing_iterable))
```

Comment:
We should reraise this exception. It used to halt the execution of loops and generators

Reply:
I would expect some tests to fail if we don't reraise here. If everything passed then this is worth looking into further